torchinductor/ir.py

```diff
@@ -1515,7 +1515,9 @@ def should_allocate(self):
 
 
 class MatrixMultiply(ExternKernelOut):
-    kernel = "aten.mm.out"
+    kernel = "aten.mm.out"    
+    if config.use_triton_mm:
```

Comment:
1) we should check this dynamically (this current version runs at import time)
2) check for the device.type=="cuda" to filter out CPU use cases